.github/workflows/conformance-ipsec-e2e.yaml

```diff
@@ -129,7 +129,7 @@ jobs:
       job_name: 'Setup & Test'
     strategy:
       fail-fast: false
-      max-parallel: 16
+      max-parallel: 100
```

Comment:
Thinking out loud: if this value would've been too high we would've already noticed failures/timeouts in this CI run, given the PR is from cilium/cilium, right?

Reply:
Yes. But also note that we currently only run 18 jobs in parallel anyway.